pkg/kubelet/volume_host.go

```diff
@@ -257,11 +257,21 @@ func (kvh *kubeletVolumeHost) GetNodeAllocatable() (v1.ResourceList, error) {
 }
 
```

Comment:
Volume Manager is needed. 
Or we can set the `not supproted` default func in `pkg/kubelet/kubelet.go`. Either is OK to me.